### PR TITLE
Fix comparison bug in Sample.Equal()

### PR DESCRIPTION
**Pull request by juliusv**

### CODE REVIEW

**Comment by juliusv** [comment]
@beorn7

File: model/value.go [hunk 1]
```diff
@@ -129,7 +129,7 @@ func (s *Sample) Equal(o *Sample) bool {
 	if !s.Timestamp.Equal(o.Timestamp) {
 		return false
 	}
-	if s.Value.Equal(o.Value) {
+	if !s.Value.Equal(o.Value) {
```
Comment (beorn7) [inline model/value.go]: Couldn't this just be
```Go
return s.Timestamp.Equal(o.Timestamp)
```
?

Comment (juliusv) [inline model/value.go]: Yup. Changed.

**Comment by beorn7** [comment]
:+1:

**Pull request merged**